Spec/RestClient.swift

```diff
@@ -565,6 +565,72 @@ class RestClient: QuickSpec {
             // RSC15
             context("Host Fallback") {
 
+                // TO3k7
+                context("fallbackHostsUseDefault option") {
+
+                    it("allows the default fallback hosts to be used when @environment@ is not production") {
+                        let options = ARTClientOptions(key: "xxxx:xxxx")
+                        options.environment = "not-production"
+                        options.fallbackHosts = ["fakeA.ably.io", "fakeB.ably.io"]
+                        options.fallbackHostsUseDefault = true
```

Comment:
It is never valid to use default and provide fallback hosts.  Why are these both set?  You are either using the defaults or using the provided fallbacks, but never both